ubg/src/pages/Room.js

```diff
@@ -267,6 +282,20 @@ function Room({setUsersData, setCurrentUser, setRoomData}) {
     });
   }
 
+  /**
+   * Sets up player to be able to play again
+   */
+  function playAgain() {
+    usersCollection.doc(user.uid).update({
+      alive: true,
+      chose: false,
+      order: Math.floor(Math.random() * 20),
+    });
+    room.update({
+      started: false,
```

Comment:
When the host starts a game doesn't that set all users to alive, chose: false?

Reply:
Only in joinRoom() does anything for individual players themselves get set. startGame() only updates for the fields in the room only. That's why it is necessary to update this here.

Reply:
Okay cool